src/main/java/de/torui/coflsky/EventRegistry.java

```diff
@@ -39,7 +39,9 @@
 import static de.torui.coflsky.EventHandler.TabMenuData;
 
 public class EventRegistry {
+	Pattern chatpattern = Pattern.compile(Configuration.getInstance().chatRegex, Pattern.CASE_INSENSITIVE);
```

Comment:
problem is that this doesn't refresh now anymore.  
It should be compiled when loaded and when `privacySettings` is called from the server